src/components/LogoIcon/LogoIcon.tsx

```diff
@@ -1,24 +1,33 @@
+import { useTheme } from '@mui/material/styles';
+
 interface Props {
-  color?: string;
   height?: number;
   width?: number;
 }
 
-const LogoIcon: React.FC<Props> = ({ height = 56, width = 55, color = '#323232' }) => (
-  <svg
-    width={width}
-    height={height}
-    viewBox="0 0 56 55"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-  >
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M30.828 4.23191C44.1933 4.23191 55.028 15.4687 55.028 29.3299C55.028 43.1912 44.1933 54.4279 30.828 54.4279C17.4627 54.4279 6.628 43.1912 6.628 29.3299C6.628 28.5501 6.66229 27.7787 6.72939 27.017C7.06049 27.0379 7.39308 27.0483 7.728 27.0483C16.8407 27.0483 24.228 19.3868 24.228 9.936C24.228 8.35107 24.0202 6.81646 23.6314 5.35987C25.9041 4.6271 28.3215 4.23191 30.828 4.23191ZM33.9567 16.7809L29.6931 16.8003L22.028 38.2246L25.3585 41.2533L36.3933 35.8895L38.2016 41.0233L41.979 39.6075L33.9567 16.7809ZM29.9362 17.4789L30.1606 18.1967L23.1775 37.7275L22.6083 37.9996L29.9362 17.4789ZM30.4037 18.8755L30.6281 19.5734L24.3174 37.1823L23.7615 37.4482L30.4037 18.8755ZM30.8525 20.2718L31.0769 20.9506L25.4525 36.6397L24.8672 36.9195L30.8525 20.2718ZM31.3202 21.6488L31.5444 22.347L26.5902 36.0957L26.0285 36.3643L31.3202 21.6488ZM31.8305 22.9356L35.011 31.9648L27.1712 35.8178L31.8305 22.9356ZM7.50351 4.63032V19.872H4.08811V4.63032H7.50351ZM3.4154 3.40357V18.6455H0V3.40357H3.4154ZM11.4795 2.69728V17.9387H8.06435V2.69728H11.4795ZM15.456 0V15.2414H12.0404V0H15.456Z"
-      fill={color}
-    />
-  </svg>
-);
+const LogoIcon: React.FC<Props> = ({ height = 56, width = 55 }) => {
+  const theme = useTheme();
+  const color =
+    theme.palette.mode === 'dark'
+      ? theme.palette.common.white
+      : theme.palette.background.paper;
```

Comment:
This may change once we fill out the pallet but it is fine for now.  Going to approve.